apps/judicial-system/backend/src/app/modules/case/internalCase.service.ts

```diff
@@ -1223,6 +1223,12 @@ export class InternalCaseService {
       attributes: ['id', 'courtCaseNumber', 'type', 'state'],
       where: {
         type: CaseType.INDICTMENT,
+        // Make sure we don't send cases that are in deleted or other inaccessible states
```

Comment:
Do we want to be extra safe and do this in `getIndictmentCase` as well?

Reply:
I did think about that, but I'm not sure what would happen in the cases where an indictment has been sent to people's digital mailboxes, if they had a link there already to an existing case that was later deleted - whether they should still be able to link from that to the case page ? Since we can't delete things from the inbox